mne/beamformer/_lcmv.py

```diff
@@ -410,10 +419,7 @@ def _lcmv_source_power(info, forward, noise_cov, data_cov, reg=0.05,
                        label=None, picks=None, pick_ori=None, rank=None,
                        weight_norm=None, verbose=None):
     """Linearly Constrained Minimum Variance (LCMV) beamformer."""
-    if weight_norm not in [None, 'unit-noise-gain']:
-        raise ValueError('Unrecognized weight normalization option in '
-                         'weight_norm, available choices are None and '
-                         '"unit-noise-gain", got "%s".' % weight_norm)
+    _check_weight_norm(weight_norm)
```

Comment:
No! `_lcmv_source_power` does not support weight norms other than `'unit-noise-gain'`!

Reply:
This is one for the beamformer agenda :)